spec/javascripts/models/beta/iteration_spec.js

```diff
@@ -1189,10 +1289,13 @@ describe("iteration", function() {
   describe('causesOverflow', () => {
     describe('when story is feature', () => {
       describe('and have 2 points', () => {
-        const story = {
-          storyType: 'feature',
-          estimate: 2
-        }
+        let story
+        beforeEach(() => {
+          story = {
+            storyType: 'feature',
+            estimate: 2
+          }
+        })
 
         describe('and velocity is 3', () => {
           it('returns truphy', () => {
```

Comment:
`truphy` -> `truthy` (I know it wasn't you but we can improve it now :smile: )

Reply:
oooooh....